subprojects/gtest.wrap

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
BTW, the directory name, `subprojects`, looks a little weird to me. Do you think we can have more meaningful and specific-to-Meson directory name instead of this general term `subprojects`?

Reply:
You do have the option to override this but Meson really discourages it. It's a convention across all projects, and there's a higher risk of introducing bugs from overriding that default

Reply:
Got it.